timescaledb/overview/release-notes/index.md

```diff
@@ -5,6 +5,10 @@ Plans](#future-plans) section. Interested in learning more about what's already
 available? Jump to [What's New](#what-x27-s-new-in-timescaledb-2-5) to see
 what's been released.
 
+<highlight type="note">
+Want to stay up-to-date with our releases? Subscribe to our [release notes email updates](https://www.timescale.com/signup/release-notes).
```

Comment:
```suggestion
Want to stay up-to-date with new releases? Subscribe to get [release notes email updates](https://www.timescale.com/signup/release-notes).
```

Reply:
Suggestion:  Subscribe to [release notes email updates](https://www.timescale.com/signup/release-notes), for the latest releases.

Reply:
Isn't that a bit repetitive though, since the first sentence says new releases also?

Reply:
Can you just make a suggestion using the `Add a suggestion` button? That would make things a lot clearer.